src/cloud-api-adaptor/pkg/paths/paths.go

```diff
@@ -0,0 +1,11 @@
+package paths
+
+const (
+	AACfgPath          = "/run/peerpod/aa.toml"
+	AuthFilePath       = "/run/peerpod/auth.json"
+	CDHCfgPath         = "/run/peerpod/cdh.toml"
+	InitDataPath       = "/run/peerpod/initdata"
+	AgentCfgPath       = "/run/peerpod/agent-config.toml"
+	ForwarderCfgPath   = "/run/peerpod/daemon.json"
+	DockerUserDataPath = "/peerpod/userdata.json"
```

Comment:
Is there an opportunity to move the existing constants to use this instead e.g. in cloud.go. I'm slightly concerned about us now having two copies that could go out of sync?

Reply:
I mean these: https://github.com/confidential-containers/cloud-api-adaptor/blob/a22b492ebc709cac8b361e850647e67494d7b973/src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go#L38-L41. They are in the cloud-api-adaptor module, so I don't think there is a dependency problem?

Reply:
If we aren't using them apart from here can we just delete them?

Reply:
that's true. we are probably using them when creating the user-data file, I probably mixed something up, in theory those should be able to use the paths module